tls/test_tls_basic.py

```diff
@@ -53,22 +40,99 @@ def start_all(self):
         self.assertTrue(deproxy_srv.wait_for_connections(timeout=1),
                         "No connection from Tempesta to backends")
 
+class TlsBasic(TlsBasicBase):
+
+    backends = [
+        {
+            'id' : 'deproxy',
+            'type' : 'deproxy',
+            'port' : '8000',
+            'response' : 'static',
+            'response_content' :
+                'HTTP/1.1 200 OK\r\n' \
+                'Content-Length: 20000\r\n' \
+                'Connection: keep-alive\r\n\r\n'
+                + 'x' * 20000
+        }
+    ]
+
     def test_bad_request(self):
         self.start_all()
         client = self.get_client('deproxy')
         client.make_request('GET / HTxTP/1.1\nHost: localhost\n\n')
-        res = client.wait_for_response(timeout=1)
+        res = client.wait_for_response(timeout=2)
         self.assertTrue(res, "Cannot process request")
         status = client.last_response.status
         self.assertEqual(status, '400', "Wrong response status: %s" % status)
+        time.sleep(1)
```

Comment:
Request contains buggy string, connection must be closed immediately after response is received. Why do we need to wait here?